students/runtests.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+"""Fw for running tests."""
+import glob
+import importlib
+import inspect
+import os
+import sys
+import traceback
+
+PATTERN = '/test_*.py'
+
+
+def select_path(path):
+    """Returns collected test files."""
+    if not path:
+        path = os.getcwd()
+        tests = glob.glob(path + PATTERN)
+        return tests
+    sys.path.append(path)
+    tests = glob.glob(path + PATTERN)
+
+    if not tests:
+        print('no tests here')  # noqa: T001

```

Comment:
I would say that this line does not belong here. It would be better to check this value at the caller level.